src/Maker/MakeCrud.php

```diff
@@ -187,7 +187,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 'entity_class_name' => $entityClassDetails->getShortName(),
                 'entity_twig_var_singular' => $entityTwigVarSingular,
                 'entity_identifier' => $entityDoctrineDetails->getIdentifier(),
-                'entity_fields' => $entityDoctrineDetails->getDisplayFields(),
+                'entity_fields' => str_replace('_', '', $entityDoctrineDetails->getDisplayFields()),
                 'route_name' => $routeName,
```

Comment:
I think this is not best solution. We need to fix it in `Doctrine\EntityDetails`

Reply:
Just for twig template, we don't want here to have _ (to match the getter).
@sadikoff If 'getDisplayFields()' is only for templating, I'll put replace in it !
Can you confirm ?

Reply:
Yes it's for templating. But you should change it carefully. `getDisplayFields()` uses `metadata->fieldMappings` it is not simple array we will need part of this array to detect correct field types